Nag for deprecation of access to conventions (#23707)
gradle/gradle: subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc

include::sample[dir="snippets/kotlinDsl/sourceSetConvention/kotlin",files="build.gradle.kts[tags=source-set-convention]"]
====

> This technique is most commonly required for source sets that are added by language plugins that did not migrate to extensions yet.

Reviewer: 💄 
```suggestion
This technique is primarily necessary for source sets added by language plugins that have yet to be migrated to extensions.
```
